php-frontend/src/main/java/org/sonar/php/tree/impl/statement/TryStatementImpl.java

```diff
@@ -33,6 +33,7 @@
 import java.util.Iterator;
 import java.util.List;
 
+// fixme (Lena) : rename to TryStatementTreeImpl
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)